tensorflow/compiler/xla/stream_executor/rocm/rocm_blas.cc

```diff
@@ -43,6 +43,9 @@ limitations under the License.
 #include "tensorflow/compiler/xla/stream_executor/scratch_allocator.h"
 #include "tensorflow/compiler/xla/stream_executor/stream_executor.h"
 
+#include "tensorflow/core/util/determinism.h"
```

Comment:
Please use the include tensorflow/tsl/util/determinism.h
And change the BUILD dependency accordingly.

Reply:
I made those changes -- pending CI